app/crud/base.py

```diff
@@ -169,6 +169,7 @@ def create_search_expressions(
 
             if column.type.python_type == str and expression.fuzzy:  # type: ignore
                 similarity_func = func.similarity(column, expression.search_term)
+                # similarity_func = func.levenshtein(column, expression.search_term)
```

Comment:
Are these commented lines for future reference? If not, we can remove them. Otherwise, please add a description of why they might be needed later.